### PR TITLE
Fix redirect warning

### PR DESCRIPTION
**Pull request by adelevie**
This pull request depends on https://github.com/18F/micropurchase/pull/88 and fixes https://github.com/18F/micropurchase/issues/89.

Even after the fix, the warning is still reported by Brakeman. I am confident that the fix actually addresses the warning, and that now it is a false positive. I would like another set of :eyes: to confirm this.

I have added this warning to `config/brakeman.ignore`.

### CODE REVIEW

**Comment by afeld** [comment]
The diff between this and #88:

https://github.com/18F/micropurchase/compare/brakeman-security-scan...fix-redirect-warning

**Comment by adelevie** [comment]
@afeld good to go after rebase or any issues to resolve?

File: config/brakeman.ignore [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      "render_path": null,
+      "location": {
+        "type": "method",
+        "class": "AuthenticationsController",
+        "method": "create"
+      },
+      "user_input": "Authenticator.new(request.env[\"omniauth.auth\"], session).perform",
+      "confidence": "High",
+      "note": "Fixed here: https://github.com/18F/micropurchase/commit/799d74393c394e016fba5eab9df1eb638a7d2b06"
+    }
```
Comment (afeld) [inline config/brakeman.ignore]: Is this necessary, with the change to the controller?

Comment (adelevie) [inline config/brakeman.ignore]: Yes it is. The change doesn't silence the warning. The note in this file
points to the commit which protects against the (already low low risk)
warning.

**Comment by afeld** [comment]
:shipit:

**Comment by adelevie** [comment]
rebased

**Pull request merged**